pkgs/applications/editors/amp/default.nix

```diff
@@ -3,17 +3,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "amp";
-  # The latest release (0.5.2) does not compile, so we use a git snapshot instead.
-  version = "unstable-2019-06-09";
+  version = "v0.6.1";
 
   src = fetchFromGitHub {
     owner = "jmacdonald";
     repo = pname;
-    rev = "2c88e82a88ada8a5fd2620ef225192395a4533a2";
-    sha256 = "0ha1xiabq31s687gkrnszf3zc7b3sfdl79iyg5ygbc49mzvarp8c";
+    rev = "e49c71d85a17a8d17423c1ab73758a0f84bcb89a";
```

Comment:
``rev = version``?

Reply:
okay, tested locally and it builds with `rev = version`. where `version = "0.6.1";`

Reply:
I believe "0.6.1" and not "v0.6.1" is the right version string anyway; this is actually one of the rare semi-niche details of Nix that is well documented :)

Reply:
ah yeah, the code is "0.6.1" https://github.com/NixOS/nixpkgs/pull/77975/files#diff-4cffb3d20d020832568c2a4eafe65e64R6

Reply:
Ah you mean in the commit message - hmm I really wonder if that even matters. Is it worth kicking off all the CI resources just to drop a character in the commit message?

Reply:
yo sebastien, can you please check the review thingy.